aodncore/pipeline/common.py

```diff
@@ -169,6 +169,7 @@ class PipelineFilePublishType(Enum):
     __slots__ = ('_is_addition_type', '_is_deletion_type', '_is_archive_type', '_is_store_type', '_is_harvest_type')
 
     # valid for both addition and deletion types
+    UNSET = (True, True, None, None, None)
```

Comment:
I think this should be `UNSET = (None, None, None, None, None)` I've tried that and all the unittests still pass. 

If some code tried to set a publish_type to UNSET using the `PipelineFile.publish_type` setter, it would then fail the validation, and so it should. UNSET should only ever be an initial state.

Reply:
Yes, good idea. The key thing was to make sure the tuple was different from everything else, hence the 'None's, but it makes perfect sense to extend it out to all Nones now that I look at it.